model/train.py

```diff
@@ -435,18 +442,22 @@ def decode_and_calculate_acc(*, p_pred_step, p_init_cache, target, config,
             predictions.append(decode_tokens(predicted[i]))
           if j == max_predict_loops - 1 and stop == False:
```

Comment:
Nit: I should have caught this earlier, but it would be simpler to use:
```suggestion
          if j == max_predict_loops - 1 and not stop:
```
(Same with the other `stop == False` line above.)